it/__init__.py

```diff
@@ -189,7 +189,7 @@ def __str__(self):
 
 
 class EsMetricsStore:
-    VERSION = "7.6.0"
+    VERSION = os.getenv("it_es_version", "7.6.0")
```

Comment:
This allows using a different ES version when running integration tests locally.

Reply:
Good idea. I'm curious: what did you need that for?

A few comments:

 * This only applies to the metrics store, and not `DISTRIBUTIONS`: the name should reflect that
 * To follow existing convention around env vars, can we also use caps and maybe prefix the environment variable with `RALLY`?

That would give `RALLY_IT_METRICS_ES_VERSION` which is maybe a bit too much. What do you think?

Reply:
(I now realize that Rally uses lower case for `http_proxy` but everything else is upper case, such as `RALLY_HOME` and `RALLY_RUNNING_IN_DOCKER`)

Reply:
...or maybe I can leave this part as it was. It's probably not important what version of Elasticsearch is being used by metrics store.

Reply:
Reverted the change.